Psychtoolbox/PsychDemos/DrawFormattedText2Demo.m

```diff
@@ -0,0 +1,138 @@
+% DrawFormattedText2Demo
+%
+% Draws formatted text, shows how to align and center text vertically
+% and/or horizontally, how line-breaks are introduced, how to transform
+% text.
+%
+% Press any key to cycle through different demo displays.
+%
+% see also: PsychDemos, Screen DrawText?, DrawSomeTextDemo, DrawFormattedText2Demo
+
+% 09/15/16    dcn    Wrote it.
+
+try
+    AssertOpenGL;   % check PTB is functioning
```

Comment:
I see there is the global ``psych_default_colormode``, but i am afraid that wouldn't do the trick. We'd need to know what the color mode for a specific window is